cmd/experimental/kjobctl/go.mod

```diff
@@ -18,7 +18,7 @@ require (
 	k8s.io/kubectl v0.31.2
 	k8s.io/utils v0.0.0-20240902221715-702e33fdd3c3
 	sigs.k8s.io/controller-runtime v0.19.0
-	sigs.k8s.io/kueue v0.8.1
+	sigs.k8s.io/kueue v0.9.0-rc.1.0.20241025140252-9bd0fccfb464
```

Comment:
/hold for release

Reply:
I'm actually inclined to merge it as is, and bump in a separate PR once 0.9.0 is released.

Reply:
/unhold

Reply:
/hold cancel
Let me open an issue to remember about bumping it: https://github.com/kubernetes-sigs/kueue/issues/3385